modules/docker/Makefile.build

```diff
@@ -16,3 +16,25 @@ endif
 	done`; \
 	echo "Building $(DOCKER_IMAGE_NAME) from $(DOCKER_FILE) with [$(DOCKER_BUILD_FLAGS)] build args..."; \
 	$(DOCKER) build $(DOCKER_BUILD_FLAGS) $$BUILD_ARGS -t $(DOCKER_IMAGE_NAME) -f $(DOCKER_FILE) $(DOCKER_BUILD_PATH)
+
+
+## Promote $IMAGE to $TARGET_IMAGE
+docker/image/promote/local:
```

Comment:
I don't think `promote` is an accurate name here since it doesn't actually `push`. Thus perhaps we should name the targets more about what they do. 

So, this might be better called `docker/image/tag`

Reply:
Maybe `docker/image/retag` would be more descriptive of the use case, and thus closer to self-documenting.

Reply:
Yea, `retag` makes sense to me.